src/styles/_catalog.scss

```
@@ -14,9 +13,17 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
+    transition-property: color, transform;
+
   }
 
   &__card:not(:nth-child(4n)) {
```

```suggestion
  &:not(:nth-child(4n)) {
```
write inside &__card